### PR TITLE
Add config for tab sizing

### PR DESCRIPTION
**Pull request by 2426021684**
Adds config for tab sizing (even vs minimum). I wrote tests but was not sure how to run them. I ran coffeelint.

### CODE REVIEW

**Comment by 50Wliu** [comment]
@2426021684 `atom --test spec`

**Comment by 2426021684** [comment]
Tests now pass

**Comment by 50Wliu** [comment]
Looks like you accidentally committed `atom.deb`.

In addition I would include an "auto" option which would use the layoutMode setting.

File: styles/config.less [hunk 1]
```diff
@@ -129,3 +114,29 @@ html { font-size: @font-size; }
...
 }
+
+// Small tabs mode ----------------------------------------------
+
+@theme-tabsizing: ~'theme-@{ui-theme-name}-tabsizing';
+
+[@{theme-tabsizing}="minimum"] {
+    .minimum-width-tabs();
+}
+
```
Comment (simurai) [inline styles/config.less]: I like @50Wliu suggestion of adding an `auto` option. Then it behaves like it currently does. Minimum in a small window, even in a larger window. Adding a media query like below should work:

```less
@media (max-width: 1280px), (max-height: 800px) {
  [@{theme-tabsizing}="auto"] {
    .minimum-width-tabs();
  }
}
```

**Comment by 2426021684** [comment]
Will fix tonight

**Comment by 2426021684** [comment]
Added auto mode and removed atom.deb. It passes all tests and coffeelint has no errors or warnings.

**Comment by simurai** [comment]
Looks good.. :ship: 

tabs

@2426021684 Thanks! :bow:

**Pull request merged**